pkg/crd-schema-gen.go

```diff
@@ -0,0 +1,3 @@
+package pkg
+
+import _ "github.com/openshift/library-go/cmd/crd-schema-gen/generator"
```

Comment:
I don't understand this change, you're running the command fro inside the library-go vendor in later commit, what do we need this for?

Reply:
this is to get glide to pick up the transitive dependencies needed for that script

Reply:
thanks @damemi